components/lib/crash/src/main/java/mozilla/components/lib/crash/service/MozillaSocorroService.kt

```diff
@@ -277,6 +286,35 @@ class MozillaSocorroService(
         return map
     }
 
+    @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)
+    internal fun readExtrasFromFile(file: File): HashMap<String, String> {
+        var resultMap = HashMap<String, String>()
+        var notJson = false
+
+        try {
+            FileReader(file).use { fileReader ->
+                val input = fileReader.readLines().firstOrNull()
+                input ?: throw JSONException("failed to read json file")
```

Comment:
nit: You can also write this as one statement:
```kotlin
val input = fileReader.readLines().firstOrNull()
    ?: throw JSONException("failed to read json file")
```

Reply:
Nice, thanks